Jenkinsfile

```diff
@@ -27,8 +27,8 @@ ansiColor('gnome-terminal') {
         } finally {
             junit(allowEmptyResults: true, testResults: 'target/test-reports/*.xml')
             junit(allowEmptyResults: true, testResults: 'tests/integration/target/test-reports/*.xml')
-            archive includes: "*sandboxes.tar.gz"
-            archive includes: "*log.tar.gz"
+            archiveArtifacts "*sandboxes.tar.gz"
+            archiveArtifacts "*log.tar.gz"
```

Comment:
I have seen the `archive includes:` to be not working. It prints something similar to:
([here is a sample build](https://jenkins.mesosphere.com/service/jenkins/view/Marathon/job/system-integration-tests/job/metronome-si-dcos-open/job/master/527/console))
```log
The archive step is deprecated, please use archiveArtifacts instead."
No files found to archive for pattern "diagnostics.zip", continuing.
```

Changing it to archiveArtifacts worked ([proof](https://jenkins.mesosphere.com/service/jenkins/blue/organizations/jenkins/system-integration-tests%2FMetronome%20SI%20pull%20request/detail/Metronome%20SI%20pull%20request/4/artifacts/)) though. I would imagine `deprecated` means the feature should still work but not sure if I am missing something.